website/versioned_sidebars/version-5.x-sidebars.json

```diff
@@ -210,7 +210,7 @@
     },
     {
       "type": "category",
-      "label": "Server Side Rendering (Beta)",
+      "label": "Server Side Rendering (New)",
```

Comment:
The word Beta deterred a lot of people from using Layout Engine. Although this is somewhat intentional, I think that the word beta makes people more nervous than they need to be when trying out the layout engine and ssr